publiccode.yaml

```diff
@@ -28,19 +28,18 @@ description:
       - Ontsluit formulieren via een API
     genericName: Formulieren component
     longDescription: >
-      Met Open Formulieren kunnen beheerders snel slimme formulieren realiseren die 
-      ontsloten worden middels een API. Met de Open Formulieren JavaScript `SDK`_ 
-      kunnen gepersonaliseerde formulieren opgehaald en getoond worden aan de 
-      gebruiker, in de stijl van de gemeente. Hierbij zijn velden vooringevuld indien 
-      mogelijk en zijn er keuzes beschikbaar afhankelijk van de gebruiker, mits deze 
-      is geauthenticeerd.
+      Met Open Formulieren kunnen beheerders snel slimme formulieren realiseren die  ontsloten
```

Comment:
the formatting changes are because of `prettier` which is hooked up in my editor :grimacing: